Structure_Engine/Query/TorsionalConstant.cs

```diff
@@ -385,7 +448,35 @@ private static double InscribedDiameterLJunction(double tw, double tf, double r)
 
         /***************************************************/
 
-        [Description("Emperical formula used to correct the torsional constant with enhancement from a T-junction. Taken from 'P385 Design of steel beams in torsion', Appendix B.")]
+        [Description("Diameter of a circle inscribed in a tapered flange T-junction connection such as a taper flange I-section. Taken from Johnston & El Darwish, 1964, Figure 6 Case 2")]
+        [Input("tw", "Web thickness, assumed to be the stem of the T.", typeof(Length))]
+        [Input("t3", "Flange thickness at theoretical intersection of flange and web centerline.", typeof(Length))]
+        [Input("r", "Root radius, assumed to be the same on both sides of the T.", typeof(Length))]
+        [Input("s", "Flange taper angle.", typeof(Length))]
+        private static double InscribedDiameterTaperTJunction(double tw, double t3, double r, double s)
+        {
+            //Equation 24
+            double f = r * s * (Math.Sqrt((1 / Math.Pow(s, 2)) + 1) - 1 - tw / (2 * r));
```

Comment:
good to check for s == 0 here. For that case we could just return the non taper inscribed diameter